crates/sui/src/client_ptb/builder.rs

```diff
@@ -965,8 +965,10 @@ impl<'a> PTBBuilder<'a> {
                     )
                     .map_err(|e| err!(pkg_loc, "{e}"))?;
                 }
-                let (dependencies, compiled_modules, _, _) =
-                    compile_result.map_err(|e| err!(pkg_loc, "{e}"))?;
+                let compiled_package = compile_result.map_err(|e| err!(pkg_loc, "{e}"))?;
+                let compiled_modules = compiled_package
+                    .get_package_bytes(false /* with_unpublished_dependencies */);
```

Comment:
We set this `with_unpublished_deps` arg to `false` above, so copied that here as well.